website/docs/language/expressions/types.mdx

```diff
@@ -80,6 +80,9 @@ List literals can be split into multiple lines for readability, but always
 require a comma between values. A comma after the final value is allowed,
 but not required. Values in a list can be arbitrary expressions.
 
+Tuples do allow elements to have mixed data types, like `["a", 15, true]`. 
```

Comment:
Perhaps better to link to the "Type Constraints" page which describes the types in more detail, or at least use the same verbiage to keep things consistent in the documentation.